src/main/java/winstone/Launcher.java

```diff
@@ -461,27 +462,23 @@ protected static void deployEmbeddedWarfile(Map<String, String> args) throws IOE
     @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN", justification = "false positive, args come from command line")
     public static void initLogger(Map<String, String> args) throws IOException {
         // Reset the log level
-        int logLevel = Option.intArg(args, Option.DEBUG.name, Logger.INFO.intValue());
+        int logLevel = Option.DEBUG.get(args);
         // boolean showThrowingLineNo = Option.LOG_THROWING_LINE_NO.get(args);
         boolean showThrowingThread = Option.LOG_THROWING_THREAD.get(args);
-        OutputStream logStream;
-        Charset logCharset;
         if (args.get("logfile") != null) {
+            Path logPath;
             try {
-                logStream = Files.newOutputStream(Paths.get(args.get("logfile")), StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+                logPath = Paths.get(args.get("logfile"));
             } catch (InvalidPathException e) {
                 throw new IOException(e);
             }
-            logCharset = StandardCharsets.UTF_8;
-        } else if (Option.booleanArg(args, "logToStdErr", false)) {
-            logStream = System.err;
-            logCharset = Charset.defaultCharset();
-        } else {
-            logStream = System.out;
-            logCharset = Charset.defaultCharset();
+            OutputStream outputStream = Files.newOutputStream(logPath, StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+            // TODO: Ideally, should change this to UTF-8, but this could cause problems for Windows users when appending to existing logs.
+            PrintStream printStream = new PrintStream(outputStream, false, Charset.defaultCharset());
+            System.setOut(printStream);
+            System.setErr(printStream);
```

Comment:
Key change in semantics: redirecting both standard streams when `--logfile` is specified. This was always the behavior in Jenkins' `--logfile` but I suspect it was not the case in (pre-Jetty) Winstone.